content/tutorial-deep-reinforcement-learning-with-pong-from-pixels.md

```diff
@@ -0,0 +1,820 @@
+
+- One of the reasons why video games are popular in deep RL research is that, unlike real-world experiments, such as RL with [remote-controlled helicopters](http://heli.stanford.edu/papers/nips06-aerobatichelicopter.pdf) ([Pieter Abbeel](https://www2.eecs.berkeley.edu/Faculty/Homepages/abbeel.html)  et al, 2006), virtual simulations can offer safer testing environments. 
+
+- If you're interested in learning about the implications of deep RL on other fields, such as neuroscience, you can refer to a [paper](https://arxiv.org/pdf/2007.03750.pdf) by [Matthew Botvinick](https://www.youtube.com/watch?v=b0LddBiF5jM) et al (2020).
+
++++ {"id": "6Haw66uARfvp"}
+
+### How to set up video playback in your Jupyter notebook
+
+- If you're using [**Binder**](https://mybinder.org) — a free Jupyter notebook-based tool — you can set up the Docker image and add `freeglut3-dev`, `xvfb`, and `x11-utils` to the `apt.txt` configuration file to install the initial dependencies. Then, to `binder/environment.yml` under `channels`, add `gym`, `pyvirtualdisplay` and anything else you may need, such as `python=3.7`, `pip`, and `jupyterlab`. Check the following [post](https://towardsdatascience.com/rendering-openai-gym-envs-on-binder-and-google-colab-536f99391cc7) for more information.
+
+- If you're using [**Google Colaboratory**](https://colab.research.google.com/notebooks/intro.ipynb) (another free Jupyter notebook-based tool), you can enable video playback of the game environments installing and setting up [X virtual frame buffer](https://en.wikipedia.org/wiki/Xvfb)/[Xvfb](https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml), [X11](https://en.wikipedia.org/wiki/X_Window_System), [FFmpeg](https://ffmpeg.org), [PyVirtualDisplay](https://github.com/ponty/PyVirtualDisplay), [PyOpenGL](http://pyopengl.sourceforge.net), and other dependencies, as described further below.
+
+1. If you're using Google Colaboratory, run the following commands in the notebook cells to help with video playback:
+
+    ```shell
+    # Install Xvfb and X11 dependencies.
+    !apt-get install -y xvfb x11-utils > /dev/null 2>&1
+    # To work with videos, install FFmpeg.
+    !apt-get install -y ffmpeg > /dev/null 2>&1
+    # Install PyVirtualDisplay for visual feedback and other libraries/dependencies.
+    !pip install pyvirtualdisplay PyOpenGL PyOpenGL-accelerate > /dev/null 2>&1
+    ```
+
+2. Then, add this Python code:
+
+    ```py
```

Comment:
Sphinx doesn't know how to render this, so we might just leave it as raw text.

```suggestion
    ```
```

Reply:
This should work if you change `py`->`python`.

Reply:
I tried that, but got 
```
WARNING: Could not lex literal_block as "python". Highlighting skipped.
```

Not sure why...

Reply:
> Sphinx doesn't know how to render this, so we might just leave it as raw text.

Following the code without the colors is kind of challenging IMHO @melissawm @rossbar  But we can fix it with a separate PR once we know what to do